error TS2307: Cannot find module 'pdfmake/build/pdfMake'

On a Linux system, `npm install` and `npm start` give the following error

```
ERROR in src/app/event-details/event-details.component.ts(12,21): error TS2307: Cannot find module 'pdfmake/build/pdfMake'.
src/app/event-public-report/event-public-report.component.ts(3,21): error TS2307: Cannot find module 'pdfmake/build/pdfMake'.
src/app/search-results-summary-report/search-results-summary-report.component.ts(3,21): error TS2307: Cannot find module 'pdfmake/build/pdfMake'.
```

The name of the file being imported is `pdfmake/build/pdfmake.js`, all lowercase. This seems to be a situation where on filesystems that are case-insensitive, the import works, but on case-sensitive filesystem, the import fails.

I'll submit a PR with the fix.

--- a/src/app/event-details/event-details.component.ts
+++ b/src/app/event-details/event-details.component.ts
@@ -9,7 +9,7 @@ import { animate, state, style, transition, trigger } from '@angular/animations'
 
 import * as L from 'leaflet';
 import * as esri from 'esri-leaflet';
-import pdfMake from 'pdfmake/build/pdfMake';
+import pdfMake from 'pdfmake/build/pdfmake';
 import pdfFonts from 'pdfmake/build/vfs_fonts';
 pdfMake.vfs = pdfFonts.pdfMake.vfs;
 import html2canvas from 'html2canvas';
@@ -504,7 +504,7 @@ export class EventDetailsComponent implements OnInit {
 
     const grayscale = esri.basemapLayer('Gray');
     const streets = esri.basemapLayer('Streets');
-    
+
     this.map = new L.Map('eventDetailsMap', {
       center: new L.LatLng(39.8283, -98.5795),
       zoom: 4,

--- a/src/app/event-public-report/event-public-report.component.ts
+++ b/src/app/event-public-report/event-public-report.component.ts
@@ -1,6 +1,6 @@
 import { Component, OnInit, AfterViewInit, ViewChild, Input, Output, EventEmitter, HostListener } from '@angular/core';
 import { Inject } from '@angular/core';
-import pdfMake from 'pdfmake/build/pdfMake';
+import pdfMake from 'pdfmake/build/pdfmake';
 import pdfFonts from 'pdfmake/build/vfs_fonts';
 import * as L from 'leaflet';
 import * as esri from 'esri-leaflet';
@@ -268,7 +268,7 @@ export class EventPublicReportComponent implements OnInit, AfterViewInit {
             // set the source string for each comment
             comment.source = this.eventLocationName(comment);
           }
-        }  
+        }
       }, 1000);
     }
     if (this.data.user.username) {

--- a/src/app/search-results-summary-report/search-results-summary-report.component.ts
+++ b/src/app/search-results-summary-report/search-results-summary-report.component.ts
@@ -1,6 +1,6 @@
 import { Component, OnInit } from '@angular/core';
 import { Inject } from '@angular/core';
-import pdfMake from 'pdfmake/build/pdfMake';
+import pdfMake from 'pdfmake/build/pdfmake';
 import pdfFonts from 'pdfmake/build/vfs_fonts';
 import { timer } from 'rxjs';
 import * as L from 'leaflet';

